Dockerfile

```diff
@@ -39,4 +39,4 @@ COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 
 ENTRYPOINT ["/usr/bin/neo-go"]
 
-CMD ["neo-go", "--config-path", "./config", "--testnet"]
+CMD ["neo-go", "--testnet"]
```

Comment:
This one should be OK, given that we're installing into the `/usr/bin` (but subject to change in #326).

Reply:
Or not. See above:
```
COPY --from=builder /neo-go/config   /config
```
So this one actually should be
```
 CMD ["neo-go", "--config-path", "/config", "--testnet"]
```
It just won't work out of the box otherwise.

Reply:
Actually checked locally:
CMD ["neo-go", "--testnet"]
should be
CMD ["node", "--testnet"]

and even "/config" and "./config" (which is default) will do the same thing

Reply:
> should be
> CMD ["node", "--testnet"]

Shouldn't be the case after the a3890910f2fdc83fb5807fcaaa30d070bcae86d6.

> and even "/config" and "./config" (which is default) will do the same thing

Maybe it's running it right from the `/`? Anyway, using `/config` seems to be more correct to me because that's the path we're installing our configs into in this case.

Reply:
> Shouldn't be the case after the a389091.

Actually, you're right here as we have parameters defined in the `CMD` and the first one of them should be `node`. Would be nice to fix it in a separate commit with a reference to my commit that broke it.

Reply:
True. agreed with having `"--config-path", "/config"`